### PR TITLE
Fix a bunch of style offences

### PR DESCRIPTION
**Pull request by Koronen**
Most of these were low-hanging fruit, fixable by using RuboCop's auto-correct feature.

### CODE REVIEW

File: lib/bundler/gem_helper.rb [hunk 1]
```diff
@@ -83,7 +83,7 @@ def build_gem
 
     def install_gem(built_gem_path = nil, local = false)
       built_gem_path ||= build_gem
-      out, _ = sh_with_code("gem install '#{built_gem_path}'#{" --local" if local}")
+      out, _code = sh_with_code("gem install '#{built_gem_path}'#{" --local" if local}")
```
Comment (segiddins) [inline lib/bundler/gem_helper.rb]: this causes a warning on 1.9.3, please keep this as `_`

**Comment by segiddins** [comment]
I'm :+1: on this other than the one comment, which is causing CI to fail

File: lib/bundler/gem_helper.rb (hunk 1 above)
Comment (Koronen) [inline lib/bundler/gem_helper.rb]: Since this was the only warning for `Style/TrailingUnderscoreVariable`, I just dropped that commit for now.

**Comment by segiddins** [comment]
@homu r+

**Comment by segiddins** [comment]
@homu r+

**Pull request merged**